qa/tasks/cephfs/test_nfs.py

```diff
@@ -14,7 +15,7 @@
 NFS_POOL_NAME = '.nfs'  # should match mgr_module.py
 
 # TODO Add test for cluster update when ganesha can be deployed on multiple ports.
-class TestNFS(MgrTestCase):
+class TestNFS(MgrTestCase, RunCephCmd):
```

Comment:
at least a different error now
```
2023-08-08T06:47:14.895 INFO:tasks.cephfs_test_runner:ERROR: test_cluster_info (tasks.cephfs.test_nfs.TestNFS)
2023-08-08T06:47:14.895 INFO:tasks.cephfs_test_runner:----------------------------------------------------------------------
2023-08-08T06:47:14.895 INFO:tasks.cephfs_test_runner:Traceback (most recent call last):
2023-08-08T06:47:14.896 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_ceph-c_b438b973c217f09b2e276d43fa768d437079c176/qa/tasks/cephfs/test_nfs.py", line 579, in test_cluster_info
2023-08-08T06:47:14.896 INFO:tasks.cephfs_test_runner:    self._test_create_cluster()
2023-08-08T06:47:14.896 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_ceph-c_b438b973c217f09b2e276d43fa768d437079c176/qa/tasks/cephfs/test_nfs.py", line 157, in _test_create_cluster
2023-08-08T06:47:14.896 INFO:tasks.cephfs_test_runner:    cluster_create = self._nfs_complete_cmd(
2023-08-08T06:47:14.896 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_ceph-c_b438b973c217f09b2e276d43fa768d437079c176/qa/tasks/cephfs/test_nfs.py", line 26, in _nfs_complete_cmd
2023-08-08T06:47:14.897 INFO:tasks.cephfs_test_runner:    return self.run_ceph_cmd(args=f"nfs {cmd}", stdout=StringIO(),
2023-08-08T06:47:14.897 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_ceph-c_b438b973c217f09b2e276d43fa768d437079c176/qa/tasks/cephfs/cephfs_test_case.py", line 58, in run_ceph_cmd
2023-08-08T06:47:14.897 INFO:tasks.cephfs_test_runner:    return self.mon_manager.run_cluster_cmd(**kwargs)
2023-08-08T06:47:14.897 INFO:tasks.cephfs_test_runner:AttributeError: 'NoneType' object has no attribute 'run_cluster_cmd'
```

Reply:
https://pulpito.ceph.com/adking-2023-08-04_21:56:32-orch:cephadm-wip-adk-testing-2023-08-04-1524-distro-default-smithi/7361645/

Reply:
This class will either need to have it's own version of `_init_mon_manager` or we'll need it to inherit from `CephFSTestCase` directly I think

Reply:
or just revert to old behavior if those don't work out anyway

Reply:
> This class will either need to have it's own version of `_init_mon_manager` or we'll need it to inherit from `CephFSTestCase` directly I think

trying this option now